src/utils/ucc_malloc.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright (C) Mellanox Technologies Ltd. 2001-2020.  ALL RIGHTS RESERVED.
+ * See file LICENSE for terms.
+ */
+
+#ifndef UCC_MALLOC_H_
+#define UCC_MALLOC_H_
+
+#include "config.h"
+
+#define ucc_malloc malloc
```

Comment:
@vspetrov Do we expect to use ucs for the definitions of these functions?

Reply:
If/when ucs exposes ucs/debug/memtrack.h as public header, we can switch to ucs_malloc/calloc/realloc. This is why i suggest we use UCC level define.

Reply:
Agree.